### ISSUE TITLE
Dropping the draggable on invalid drop zone

### ISSUE THREAD

**Issue by shidhincr** [caitp/angular-drop#22]
Can we've a feature that when a draggable element is dropped on an invalid drop zone, the element can move back to it's previous place ?

I'm not sure if this is already implemented.. If not consider this as a feature request.

**Comment by adam77** [comment]
Are we ok with an angular 1.4 (new animation engine) solution to this?

**Comment by caitp** [comment]
Absolutely, although breaking changes creep into ngAnimate pretty often

**Comment by adam77** [comment]
possible (mostly) CSS solution:
    - for statically positioned elements, use relative positioning during the drag process (so they have somewhere to return; note jq-ui uses relative)
    - on (invalid) dragEnd add style `transition: top .5s, left .5s`
    - on (invalid) dragEnd set style.top,style.left to 0 (or initial values for pre-drag positioned elements)
    - on transitionEnd (css event) reset initial style.position; remove transition

notes:
    * won't work on ie9 (the transition could be considered example of progressive enhancement and therefore not essential; ie9 support could be added at a later date)
    * this solution doesn't make use of ngAnimate
